docs/openwrt_compile_source/Makefile

```diff
@@ -0,0 +1,59 @@
+include $(TOPDIR)/rules.mk
```

Comment:
Should the scripts to compile our package from source be located under the docs/ folder? It may be better to place the compiling scripts themselves under setup

Reply:
My thought process is that the src and setup folders contain the core code for tinyHIPPO and those files will be installed onto the router during the install process. I think that Makefile and script should stay in the docs because it would only be used in the event that someone wants to compile tinyHIPPO themselves, develop some additions, and test it out.